components/Transaction/InvoiceModal.tsx

```diff
@@ -0,0 +1,239 @@
+import React, { useState, useEffect, ReactElement } from 'react'
+import style from './transaction.module.css'
+import Button from 'components/Button'
+import { CreateInvoicePOSTParameters } from 'utils/validators'
+import axios from 'axios'
+
+export interface InvoiceData {
+  id?: string
+  invoiceNumber: string
+  amount: number
```

Comment:
Not a good idea to use `number` here, can get messy with float imprecision. Best to stick to `Prisma.Decimal`

Remember to change this also in the other interfaces you're creating.